### ISSUE TITLE
Missing hourly files

### ISSUE THREAD

**Issue by janpf** [janpf/ctt#11]
TL;DR: Does anybody have the collection of hourly files for the last few days?

Explanation why I'm missing those: my script didn't catch them, as I'm not archiving the hourly keys daily and the server stopped providing the hourly files for all days but yesterday:

available dates:
[svc90.main.px.t-online.de/version/v1/diagnosis-keys/country/DE/date](http://svc90.main.px.t-online.de/version/v1/diagnosis-keys/country/DE/date)

hourly files available for yesterday (all hourly files available):
[svc90.main.px.t-online.de/version/v1/diagnosis-keys/country/DE/date/2021-01-29/hour](http://svc90.main.px.t-online.de/version/v1/diagnosis-keys/country/DE/date/2021-01-29/hour)

the day before (no longer available):
[svc90.main.px.t-online.de/version/v1/diagnosis-keys/country/DE/date/2021-01-28/hour](http://svc90.main.px.t-online.de/version/v1/diagnosis-keys/country/DE/date/2021-01-28/hour)

Thanks :)
